src/test/java/net/datafaker/providers/base/PlanetTest.java

```diff
@@ -0,0 +1,199 @@
+package net.datafaker.providers.base;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+import java.util.List;
+
+import static net.datafaker.providers.base.Planet.PlanetName.EARTH;
+import static net.datafaker.providers.base.Planet.PlanetName.JUPITER;
+import static net.datafaker.providers.base.Planet.PlanetName.MARS;
+import static net.datafaker.providers.base.Planet.PlanetName.MERCURY;
+import static net.datafaker.providers.base.Planet.PlanetName.NEPTUNE;
+import static net.datafaker.providers.base.Planet.PlanetName.SATURN;
+import static net.datafaker.providers.base.Planet.PlanetName.URANUS;
+import static net.datafaker.providers.base.Planet.PlanetName.VENUS;
+import static net.datafaker.providers.base.Planet.PlanetName.PLUTO;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
it's better to use assertions from assertj only to avoid potential issues while using mix of both